tabs/tab_data.py

```
@@ -181,97 +179,138 @@ def _():
                 'ICC_SysBP_Rater2': {'type': 'Continuous', 'label': 'Sys BP (Rater 2)', 'map': {}},
             }
             
+            return new_df, meta
+            
+        except Exception as e:
+            logger.error(f"Error generating example data: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)